sdk/eventhub/Azure.Messaging.EventHubs/assets/samples-azure-deploy.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Are those tabs I see!? 😢

Reply:
Good god, not intentionally.  This is what I get for stealing the track one template.   Fixed!

Reply:
Phew! Panic attack adverted. 🤣